examples/applications/plot_topics_extraction_with_nmf_lda.py

```diff
@@ -1,8 +1,7 @@
 """
-========================================================
-Topic extraction with Non-negative Matrix Factorization\
-and Latent Dirichlet Allocation
-========================================================
+=======================================================================================
```

Comment:
flake8 calls pep8, right? ;) but yes, I would also say it doesn't violate PEP 8, which is why I left it as is. Should I do a ``#noqa``?